lib/zendesk_apps_tools/deploy.rb

```diff
@@ -62,26 +62,27 @@ def upload(path)
       say_error_and_exit e.message
     end
 
-    def find_app_id
+    def find_app_id(product_name = 'v2') # use the v2 endpoint if no product name is provided
```

Comment:
What are products in this context? Might be my lack of knowledge of ZAT, but this looks like a framework version to me. If that's the case, maybe call that variable `framework_version`?

Reply:
Products are `support`, `chat` etc. Technically those references are to a `v3` endpoint. We fallback to the `v2` endpoint if no product is specified as the response there helpfully looks identical to the v3 ones